### PR TITLE
Landing page

### PR DESCRIPTION
**Pull request by theprogrammerinyou**
The landing page as well as the login page.

### CODE REVIEW

File: login/login.css [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+    width: 100%;
+    padding: 1rem;
+    background-color: var(--primary-color);
+    border: none;
+    cursor: pointer;
+}
+
+.card {
+    background-color: var(--white);
+    width: 400px;
```
Comment (GuptaEesh) [inline login/login.css:11]: convert these to rem as these are very large values.

**Review by GuptaEesh — COMMENTED** [review]
- Color variables used
- % values used for responsiveness.
- just added few suggestions.
Otherwise good to merge.

File: login/login.html [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+            />
+          </div>
+        </div>
+      </nav>
+    </main>
+    <section class="main-section">
+      <div class="card basic-margin-padding">
+        <h3 class="title-section">Login</h3>
+        <div class="label-inputs">
+          <label for="email" class="inputLabel">Email address *</label>
```
Comment (prachi-sahani) [inline login/login.html:47]: - better naming convention can be used(input-label and inputLabel can be confusing)
- be consistent in naming classes(camelCase is not being used at other places in the code)

File: login/login.html [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+          <input
+            class="input"
+            id="email"
+            type="text"
```
Comment (prachi-sahani) [inline login/login.html:51]: can be of type email - optional

**Review by prachi-sahani — COMMENTED** [review]
Apart from the mentioned points the code seems fine

**Pull request merged**